docs/FAQ.rst

```diff
@@ -83,19 +83,20 @@ Also see https://software.intel.com/en-us/articles/python-mpi4py-on-intel-true-s
 **can't open hfi unit: -1 (err=23)**
 **[13] MPI startup(): tmi fabric is not available and fallback fabric is not enabled**
 
-This may occur on TMI when libEnsemble Python processes have been launched to a node and these,
-in turn, launch jobs on the node; creating too many processes for the available contexts. Note that
-while processes can share contexts, the system is confused by the fact that there are two
-phases: first libEnsemble processes and then subprocesses to run user jobs. The solution is to
-either reduce the number processes running or to specify a fallback fabric through environment
-variables::
+This may occur on TMI when libEnsemble Python processes have been launched to a
+node and these, in turn, launch tasks on the node; creating too many processes
+for the available contexts. Note that while processes can share contexts, the
+system is confused by the fact that there are two phases: first libEnsemble
+processes and then subprocesses to run user tasks. The solution is to either
+reduce the number of processes running or to specify a fallback fabric through
+environment variables::
 
     unset I_MPI_FABRICS
     export I_MPI_FABRICS_LIST=tmi,tcp
     export I_MPI_FALLBACK=1
 
 Alternatively, libEnsemble can be run in central mode where all workers run on dedicated
-nodes, while launching all sub-jobs onto other nodes.
+nodes, while launching all sub-tasks onto other nodes.
```

Comment:
Why is this "sub-tasks"? Doesn't central mode mean launching "a unit of computational work"?

Reply:
You're right, this should be "tasks".